infrastructure/api-configuration/api-server-instance-user-data.tpl.sh

```diff
@@ -16,55 +16,61 @@ cd /home/ubuntu
+    tee /etc/apt/sources.list.d/docker.list > /dev/null
+apt update -y
+apt install docker-ce docker-ce-cli -y
+
 if [[ ${stage} == "staging" || ${stage} == "prod" ]]; then
     # Check here for the cert in S3, if present install, if not run certbot.
     if [[ $(aws s3 ls "${scpca_portal_cert_bucket}" | wc -l) == "0" ]]; then
-	# Create and install SSL Certificate for the API.
-	# Only necessary on staging and prod.
-	# We cannot use ACM for this because *.bio is not a Top Level Domain that Route53 supports.
-	apt-get install -y software-properties-common
-	add-apt-repository ppa:certbot/certbot
-	apt-get update
-	apt-get install -y python-certbot-nginx
-
-	# g3w4k4t5n3s7p7v8@alexslemonade.slack.com is the email address we
-	# have configured to forward mail to the #teamcontact channel in
-	# slack. Certbot will use it for "important account
-	# notifications".
-
-	# The certbot challenge cannot be completed until the aws_lb_target_group_attachment resources are created.
-	sleep 180
-	BASE_URL="scpca.alexslemonade.org"
-	if [[ ${stage} == "staging" ]]; then
-            certbot --nginx -d api.staging.$BASE_URL -n --agree-tos --redirect -m g3w4k4t5n3s7p7v8@alexslemonade.slack.com
-	elif [[ ${stage} == "prod" ]]; then
-            certbot --nginx -d api.$BASE_URL -n --agree-tos --redirect -m g3w4k4t5n3s7p7v8@alexslemonade.slack.com
-	fi
-
-	# Add the nginx.conf file that certbot setup to the zip dir.
-	cp /etc/nginx/nginx.conf /etc/letsencrypt/
-
-	cd /etc/letsencrypt/ || exit
-	sudo zip -r ../letsencryptdir.zip "../$(basename "$PWD")"
-
-	# And then cleanup the extra copy.
-	rm /etc/letsencrypt/nginx.conf
-
-	cd - || exit
-	mv /etc/letsencryptdir.zip .
-	aws s3 cp letsencryptdir.zip "s3://${scpca_portal_cert_bucket}/"
-	rm letsencryptdir.zip
-    else
-	zip_filename=$(aws s3 ls "${scpca_portal_cert_bucket}" | head -1 | awk '{print $4}')
-	aws s3 cp "s3://${scpca_portal_cert_bucket}/$zip_filename" letsencryptdir.zip
-	unzip letsencryptdir.zip -d /etc/
-	mv /etc/letsencrypt/nginx.conf /etc/nginx/
-	service nginx restart
+        # Create and install SSL Certificate for the API.
+        # Only necessary on staging and prod.
+        # We cannot use ACM for this because *.bio is not a Top Level Domain that Route53 supports.
+        apt-get update
+        apt install certbot python3-certbot-nginx -y
+
+        # g3w4k4t5n3s7p7v8@alexslemonade.slack.com is the email address we
+        # have configured to forward mail to the #teamcontact channel in
+        # slack. Certbot will use it for "important account
+        # notifications".
+
+        # The certbot challenge cannot be completed until the aws_lb_target_group_attachment resources are created.
+        sleep 180
+        BASE_URL="scpca.alexslemonade.org"
+        if [[ ${stage} == "staging" ]]; then
+                certbot --nginx -d api.staging.$BASE_URL -n --agree-tos --redirect -m g3w4k4t5n3s7p7v8@alexslemonade.slack.com
```

Comment:
May be a good idea to declare NOTIFICATION_EMAIL="g3w4k4t5n3s7p7v8@alexslemonade.slack.com"